src/HolographicCamera.Unity/Assets/HolographicCamera/Scripts/KeepDeviceActive.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+#if UNITY_WSA && WINDOWS_UWP
+    using System;
+    using Windows.System.Display;
+#endif
+
+public class KeepDeviceActive : MonoBehaviour
+{
+#if UNITY_WSA && WINDOWS_UWP
+    private static DisplayRequest displayRequest;
+#endif
+
+    private void Start()
+    {
+        CreateDisplayRequest();
+    }
+
+    private void OnDestroy()
+    {
+#if UNITY_WSA && WINDOWS_UWP
+        if (displayRequest != null)
+        {
+            try
+            {
+                displayRequest.RequestRelease();
+                Debug.Log("Display request released.");
+            }
+            catch (Exception e)
+            {
+                Debug.LogError($"Error releasing display request: {e.Message}");
+            }
+        }
+#endif
+    }
+
+    private void CreateDisplayRequest()
+    {
+#if UNITY_WSA && WINDOWS_UWP
```

Comment:
Is it possible to use this  #if wrapper at a higher level (i.e. completely enclosing Start/OnDestroy/CreateDisplayRequest?)

The micro-managey part about the current set up is a bit inconsistent in where things get enclosed (i.e. why is ifdefing caught within this function instead of inside start, is there a lot of value to having functions that end up just being empty on non UWP, etc)